src/system.py

```diff
@@ -315,18 +315,249 @@ def __init__(self):
+                            }
+                        }
+                    },
+                    'UHF_SET_MORSE': {
+                        'subPort': 30,
+                        'inoutInfo': {
+                            'args': ['>U36'],
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'UHF_SET_MIDI': {
+                        'subPort': 31,
+                        'inoutInfo': {
+                            'args': ['>U36'],
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'UHF_SET_BEACON_MSG': {
+                        'subPort': 32,
+                        'inoutInfo': {
+                            'args': ['>u1', '>U36'],  # 120 too large
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'UHF_SET_I2C': {
+                        'subPort': 33,
+                        'inoutInfo': {
+                            'args': ['>u1'],
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'UHF_WRITE_FRAM': {
+                        'subPort': 34,
+                        'inoutInfo': {
+                            'args': ['>u4', '>U16'],
+                            'returns': {
+                                'err': '>b',
+                            }
+                        }
+                    },
+                    'UHF_SECURE': {
+                        'subPort': 35,
+                        'inoutInfo': {
+                            'args': None,
+                            'returns': {
+                                'err': '>b',
+                                'Status': '>u1',
+                            }
+                        }
+                    },
+                    'UHF_GET_FULL_STAT': {
+                        'subPort': 36,
+                        'inoutInfo': {
+                            'args': None,
+                            'returns': {
+                                'err': '>b',
+                                'RFTS': '>u1',
+                                'FRAM': '>u1',
+                                'SEC': '>u1',
+                                'CTS': '>u1',
+                                'Boot': '>u1',
+                                'Pipe': '>u1',
+                                'BCN': '>u1',
+                                'Echo': '>u1',
+                                'RF Mode1': '>u1',  # Use table 9 to improve
+                                'RF Mode2': '>u1',
+                                'RF Mode3': '>u1',
```

Comment:
The RF Mode is like `110` or `010`. However, the status control is an array[12]. I couldn't figure out how to send 3 consecutive `uint8_t` as one. So, I improvised. Also, I should check with Thomas why there are 12. The manual shows more.

Reply:
Yeah, I hadn't even thought of arrays in the return type, This should be good enough for now at least.

Reply:
Actually, I'm a bit confused about this. Perhaps you can explain at the work session